README.md

```diff
@@ -59,10 +64,19 @@ The gem has undergone two major changes: 1) it's been renamed to SolidusBraintre
 
 Additionally, you can rename any other references to SolidusPaypalBraintree in your app to SolidusBraintree. This will fix any deprecation warnings that come with SolidusBraintree 2.0.0.
 
-## Here be dragons: upgrading from SolidusBraintree 1.2.0
+### Here be dragons: upgrading from SolidusBraintree 1.2.0
 
 Considering that SolidusBraintree 2.0.0 is based on the SolidusPaypalBraintree codebase, please be warned that migrating directly from SolidusBraintree 1.2.0 to 2.0.0 would lead to breaking changes. Since SolidusBraintree 1.x was already [deprecated](https://github.com/solidusio/solidus_braintree/tree/v1.x#deprecation-notice-warning-construction), we don't provide at the moment a ready-made solution for migrating from SolidusBraintree 1.2.0 to 2.0.0.
 
+## Compatibility
+
+Here are the versions of SolidusBraintree and their compatible Solidus and frontend versions:
+
+| SolidusBraintree | Solidus | Frontend               |
+|------------------|---------|------------------------|
+| 2.0              | 3.4     | SolidusStarterFrontend |
+| 1.3              | 3.4     | SolidusFrontend        |
```

Comment:
SolidusBraintree 1.3 hasn't been released yet. Well need to update the 1.x branch and backport all the master changes up to https://github.com/solidusio/solidus_braintree/pull/99.

Reply:
FYI, added https://github.com/solidusio/solidus_braintree/issues/127.